src/cpp_generator.cpp

```diff
@@ -174,6 +174,11 @@ std::string cpp_generator::generate_point_op(const operation& op,
         else if(with_char(::isdigit)(key[0]))
         {
             auto i = std::stoul(key);
+            // For an optional argument where i >= args.size(), treat
+            // the optional argument as a straight zero. This will
+            // cacel out the optional bias, if it exists.
```

Comment:
This comment doesnt match what is done in the code.

Reply:
Nice catch.